src/ResourceManager/KeyVault/Commands.KeyVault/Commands/GetAzureKeyVaultCertificateIssuer.cs

```diff
@@ -42,17 +40,28 @@ public class GetAzureKeyVaultCertificateIssuer : KeyVaultCmdletBase
         /// Vault name
         /// </summary>
         [Parameter(Mandatory = true,
+            ParameterSetName = ByNameParameterSet,
             Position = 0,
             ValueFromPipelineByPropertyName = true,
             HelpMessage = "Vault name. Cmdlet constructs the FQDN of a vault based on the name and currently selected environment.")]
         [ValidateNotNullOrEmpty]
         public string VaultName { get; set; }
 
         /// <summary>
-        /// Name.
+        /// Vault object
         /// </summary>
         [Parameter(Mandatory = true,
-            ParameterSetName = ByNameParameterSet,
+            ParameterSetName = ByInputObjectParameterSet,
+            Position = 0,
+            ValueFromPipeline = true,
+            HelpMessage = "KeyVault object.")]
+        [ValidateNotNullOrEmpty]
+        public PSKeyVault InputObject { get; set; }
+
+        /// <summary>
+        /// Name.
+        /// </summary>
+        [Parameter(Mandatory = false,
             Position = 1,
             ValueFromPipelineByPropertyName = true,
```

Comment:
We should be removing these valuefrompipelinebypropertyname for the interactive parameter sets, right?  I assume this is happening in the breaking change update.

Reply:
Yes, this will be happening for the breaking change update.